src/mysqlserver.py

```diff
@@ -181,5 +181,4 @@ def version(self) -> Union[str, None]:
     def new_password(length: int = 16) -> str:
```

Comment:
Might be better as a free function rather than a static method, e.g. `from utils import mkpasswd`

Reply:
Yes, definitely. Thanks!
In fact all these common functions we use across charms should be in ops or in a custom shared module.

Reply:
https://github.com/canonical/operator/issues/536